src/gui/MainWindow.cpp

```diff
@@ -786,14 +783,16 @@ void MainWindow::createNewProject()
 
 void MainWindow::createNewProjectFromTemplate( QAction * _idx )
 {
-	if( m_templatesMenu != NULL && mayChangeProject(true) )
+	if( m_templatesMenu && mayChangeProject(true) )
 	{
-		QString dir_base = m_templatesMenu->actions().indexOf( _idx )
-						>= m_custom_templates_count ?
-				ConfigManager::inst()->factoryProjectsDir() :
-				ConfigManager::inst()->userProjectsDir();
+		int indexOfTemplate = m_templatesMenu->actions().indexOf( _idx );
+		bool isFactoryTemplate = indexOfTemplate >= m_custom_templates_count;
+		QString dir_base =  isFactoryTemplate ?
```

Comment:
@Wallacoloo I have changed the local variable to camel case. I think adjusting the members of `MainWindow` is something for another issue / pull request.